docs/release/release-team.md

```diff
@@ -49,8 +67,8 @@ As noted above, making changes to  the CAPI release cadence is out of scope for
 - **Release Lead**: responsible for coordinating release activities, assembling the release team, taking ultimate accountability for all release tasks to be completed on time, and ensuring that a retrospective happens. The lead is also responsible for ensuring a successor is selected and trained for future release cycles.
 - **Communications/Docs/Release Notes Manager**: Responsible for communicating key dates to the community, improving release process documentation, and polishing release notes. Also responsible for ensuring the user-facing Netlify book and provider upgrade documentation are up to date.
 - **CI Signal/Bug Triage/Automation Manager**: Assumes the responsibility of the quality gate for the release and makes sure blocking issues and bugs are triaged and dealt with in a timely fashion. Helps improve release automation and tools.
-- **Shadow**: Any Release Team member may select one or more mentees to shadow the release process in order to help fulfill future Release Team staffing requirements and continue to grow the CAPI community in general.
-
+- **Team member**: Any Release Team lead or manager may select one or more additional members to help with their the release tasks. These team members will help fulfill future Release Team staffing requirements and continue to grow the CAPI community in general.
+- **Release Writer**: responsible for tasks which require write access to the Cluster API repo including creating release tags and creating a release branch. This role must be filled by someone on the [`cluster-api-maintainers` list](https://github.com/kubernetes-sigs/cluster-api/blob/5999d1679509d4f3b09d50f394c43bc0f8d33c94/OWNERS_ALIASES#L21-L27).
```

Comment:
```suggestion
- **Release Writer**: Responsible for tasks which require write access to the Cluster API repo including creating release tags and creating a release branch. This role must be filled by someone on the [`cluster-api-maintainers` list](https://github.com/kubernetes-sigs/cluster-api/blob/5999d1679509d4f3b09d50f394c43bc0f8d33c94/OWNERS_ALIASES#L21-L27).
```
nit

I think we should link the file on main instead, but then we should drop the lines in the link